Copyedits on porting_from_freertos.tex (#63)
CHERIoT-Platform/book: text/porting_from_freertos.tex

struct RecursiveMutexState __SuspendFlagLock;
\end{cxxsnippet}

> A futex-based lock is very cheap to acquire in the uncontended case; it requires a single atomic compare-and-swap instruction, which may be a function call to a library routine that runs with interrupts disabled if the hardware does not support atomics.

Reviewer: Hmm, this is a bit wrong, since it isn't an instruction in the second case.  How about:

```suggestion
A futex-based lock is very cheap to acquire in the uncontended case; it requires a single atomic compare-and-swap instruction.
If the hardware doesn't support atomic operations then this may instead be a function call to a library routine that runs with interrupts disabled.
```

Author: Updated.
